packages/teleterm/src/ui/DocumentCluster/ClusterResources/Applications/Applications.tsx

```diff
@@ -33,52 +33,58 @@ import Table, { Cell } from 'design/DataTable';
 import * as types from 'teleterm/ui/services/clusters/types';
 import { useApps, State } from './useApps';
 import { renderLabelCell } from '../renderLabelCell';
+import { Danger } from 'design/Alert';
 
 export default function Container() {
   const state = useApps();
   return <AppList {...state} />;
 }
 
 export function AppList(props: State) {
-  const { apps = [] } = props;
+  const { apps = [], syncStatus } = props;
 
   return (
-    <StyledTable
-      data={apps}
-      columns={[
-        {
-          altKey: 'app-icon',
-          render: renderAppIcon,
-        },
-        {
-          key: 'name',
-          headerText: 'Name',
-          isSortable: true,
-        },
-        {
-          key: 'description',
-          headerText: 'Description',
-          isSortable: true,
-        },
-        {
-          key: 'publicAddr',
-          headerText: 'Address',
-          render: renderAddress,
-          isSortable: true,
-        },
-        {
-          key: 'labelsList',
-          headerText: 'Labels',
-          render: renderLabelCell,
-        },
-        {
-          altKey: 'launch-btn',
-          render: renderConnectButton,
-        },
-      ]}
-      emptyText="No Applications Found"
-      pagination={{ pageSize: 100, pagerPosition: 'bottom' }}
-    />
+    <>
+      {syncStatus.status === 'failed' && (
```

Comment:
How can I test this? In what scenario is `syncStatus` set `failed`? I tried to either disconnect my network connection or shut down the cluster, but in both cases the app crashes with

```
2 UNKNOWN: dial tcp 127.0.0.1:3023: connect: connection refused
```